build-test-elasticsearch7.xml

```diff
@@ -856,6 +857,23 @@ xpack.monitoring.collection.enabled: true</echo>
 			select="\1"
 		/>
 
+		<exec executable="curl" outputproperty="indexList">
+			<arg value="-X" />
+			<arg value="GET" />
+			<arg value="http://localhost:9200/_cat/indices?h=idx" />
+			<arg value="-s" />
+		</exec>
+
+		<propertyregex
+			input="${indexList}"
+			override="true"
+			property="indexList"
+			regexp="\n"
+			replace=","
+		/>
+
+		<echo>${indexList}</echo>
```

Comment:
Let's give a title to the things we index so it's more obvious to someone without context